specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/examples/CosmosDBManagedCassandraClusterGet.json

```diff
@@ -0,0 +1,73 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "resourceGroupName": "cassandra-prod-rg",
+    "api-version": "2020-06-01-preview",
+    "clusterName": "cassandra-prod"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid/resourceGroups/cassandra-prod-rg/providers/Microsoft.DocumentDB/cassandraClusters/cassandra-prod",
+        "name": "cassandra-prod",
+        "type": "Microsoft.DocumentDB/cassandraClusters",
+        "location": "West US",
+        "tags": {},
+        "properties": {
+          "provisioningState": "Succeeded",
+          "delegatedManagementSubnetId": "/subscriptions/536e130b-d7d6-4ac7-98a5-de20d69588d2/resourceGroups/customer-vnet-rg/providers/Microsoft.Network/virtualNetworks/customer-vnet/subnets/management",
+          "cassandraVersion": "3.11",
+          "hoursBetweenBackups": 24,
+          "authenticationMethod": "Cassandra",
+          "externalSeedNodes": [
+            {
+              "ipAddress": "10.52.221.2"
+            },
+            {
+              "ipAddress": "10.52.221.3"
+            },
+            {
+              "ipAddress": "10.52.221.4"
+            }
+          ],
+          "overrideClusterName": "ClusterNameIllegalForAzureResource",
+          "seedNodes": [
+            {
+              "ipAddress": "10.52.221.2"
+            },
+            {
+              "ipAddress": "10.52.221.3"
+            },
+            {
+              "ipAddress": "10.52.221.4"
+            },
+            {
+              "ipAddress": "192.168.12.2"
+            },
+            {
+              "ipAddress": "192.168.12.3"
+            },
+            {
+              "ipAddress": "192.168.12.4"
+            }
+          ],
+          "clientCertificates": [
+            {
+              "pem": "-----BEGIN CERTIFICATE-----\n...Base64 encoded certificate...\n-----END CERTIFICATE-----"
```

Comment:
>"-----BEGIN CERTIFICATE [](start = 21, length = 23)

if these have private keys, then they should be annotated as secrets

Reply:
They are public keys.